src/seedu/addressbook/AddressBook.java

```diff
@@ -178,21 +164,17 @@
      * ====================================================================================================
      */
 
-    /**
-     * List of all persons in the address book.
-     */
+    /** List of all persons in the address book. */
     private static final ArrayList<String[]> ALL_PERSONS = new ArrayList<>();
 
     /**
      * Stores the most recent list of persons shown to the user as a result of a user command.
      * This is a subset of the full list. Deleting persons in the pull list does not delete
      * those persons from this list.
      */
-    private static ArrayList<String[]> latestPersonListingView = getAllPersonsInAddressBook(); // initial view is of all
+    private static ArrayList<String[]> latestPersonListingView = getEveryPerson(); // initial view is of all
```

Comment:
The naming here is a bit subjective. Could you find another things to refactor? :)